### PR TITLE
feature: multi action flow

### LINKED ISSUE #1993: Implement multi action flow in Avo 2
From [here](https://discord.com/channels/740892036978442260/1162459368827146292)

### PR DESCRIPTION
**Pull request by Paul-Bob**
# Description

Fixes #1993

[multi_flux_v2.webm](https://github.com/avo-hq/avo/assets/69730720/9a0ecea9-8202-426b-a0fc-49acce9eee0d)

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the [documentation](https://github.com/avo-hq/avodocs)
  - https://github.com/avo-hq/avodocs/pull/124
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -167,3 +167,5 @@ gem "image_processing", "~> 1.12"
 gem "prefixed_ids"
 
 gem "mapkick-rb", "~> 0.1.4"
+
+gem "turbo_power", "~> 0.5.0"
```
Comment (adrianthedev) [inline Gemfile:171]: This needs to be in the `.gemspec` file as a runtime dependency

Comment (Paul-Bob) [reply Gemfile:171]: Already added on this commit https://github.com/avo-hq/avo/pull/1994/commits/9a09b677a2b53cce3ff6f89f61becd8354c1dec6

**Comment by adrianthedev** [comment]
Is this good to merge?

**Comment by Paul-Bob** [comment]
> Is this good to merge?

Yes

**Pull request merged**